examples/graphiql-create-react-app/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "example-graphiql-create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react": "^16.9.34",
+    "@types/react-dom": "^16.9.6",
+    "graphiql": "file:../../packages/graphiql",
+    "graphql": "^15.0.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
```

Comment:
if this was not added by CRA we should remove it. only CRA-related changes except adding GraphiQL component. I haven't used CRA in a few years really so I'm not sure

Reply:
Yup it was added by CRA.